packages/injected-script/sdk/index.ts

```diff
@@ -1,12 +1,14 @@
 import { CustomEventId, decodeEvent } from '../shared'
-import { onEthEvent } from './bridgedEthereum'
-import { onCoin98Event } from './bridgedCoin98'
-import { onSolanaEvent } from './bridgedSolana'
+import { onEthEvent } from './injectedEthereumProvider'
+import { onCoin98Event } from './injectedCoin98Provider'
+import { onSolanaEvent } from './injectedSolanaProvider'
 import { sendEvent, rejectPromise, resolvePromise } from './utils'
 
-export { bridgedEthereumProvider } from './bridgedEthereum'
-export { bridgedCoin98Provider } from './bridgedCoin98'
-export { bridgedSolanaProvider } from './bridgedSolana'
+export * from '../shared'
```

Comment:
please do not export everything. the surface of this package should keep small. only export things you need.

Reply:
@guanbinrui